diff -up steam/lib/udev/rules.d/60-steam-input.rules steam/lib/udev/rules.d/new-60-steam-input.rules
--- steam/lib/udev/rules.d/60-steam-input.rules 2019-04-10 18:47:29.000000000 +0200
+++ steam/lib/udev/rules.d/new-60-steam-input.rules     2019-04-20 00:59:45.318177255 +0200
@@ -72,3 +72,7 @@ KERNEL=="hidraw*", ATTRS{idVendor}=="095
 
 # NVIDIA Shield Controller (2017 - NVIDIA_Controller_v01.04 over bluetooth hidraw)
 KERNEL=="hidraw*", KERNELS=="*0955:7214*", MODE="0660", TAG+="uaccess"
+
+# This rule is necessary for gamepad emulation; make sure each user requiring
+# it is part of the input group.
+KERNEL=="uinput", MODE="0660", GROUP="input", OPTIONS+="static_node=uinput"
