diff -Naur steam/steam steam.tpg/steam
--- steam/steam	2016-03-29 20:03:30.000000000 +0000
+++ steam.tpg/steam	2016-06-29 18:54:35.878185705 +0000
@@ -186,6 +186,7 @@
 # go to the install directory and run the client
 cp "$LAUNCHSTEAMBOOTSTRAPFILE" "$LAUNCHSTEAMDIR/bootstrap.tar.xz"
 cd "$LAUNCHSTEAMDIR"
+
 if [ "`command -v tee`" != "" ]; then
 	mkdir -p --mode=777 /tmp/dumps
     if [[ -f "/tmp/dumps/${USER}_stdout.txt" ]]; then
@@ -195,3 +196,6 @@
 else
 	exec "$LAUNCHSTEAMDIR/$STEAMBOOTSTRAP" "$@"
 fi
+
+# (tpg) use our own libraries
+find "$HOME/.steam/root" \( -name "libgcc_s.so*" -o -name "libstdc++.so*" -o -name "libxcb.so*" -o -name "libgpg-error.so*" \) -print -delete
