diff -uraN steam-launcher/subprojects/steam-devices/60-steam-input.rules omv-steam-launcher/subprojects/steam-devices/60-steam-input.rules
--- steam-launcher/subprojects/steam-devices/60-steam-input.rules	2021-09-08 20:10:10.000000000 +0200
+++ omv-steam-launcher/subprojects/steam-devices/60-steam-input.rules	2021-10-03 16:13:43.604337896 +0200
@@ -125,3 +125,7 @@
 
 # Thrustmaster eSwap Pro
 KERNEL=="hidraw*", ATTRS{idVendor}=="044f", ATTRS{idProduct}=="d00e", MODE="0660", TAG+="uaccess"
+
+# This rule is necessary for gamepad emulation; make sure each user requiring
+# it is part of the input group.
+KERNEL=="uinput", MODE="0660", GROUP="input", OPTIONS+="static_node=uinput"
