diff -uraN steam-launcher/bin_steam.sh omv-steam-launcher/bin_steam.sh
--- steam-launcher/bin_steam.sh	2020-07-29 17:44:37.000000000 +0200
+++ omv-steam-launcher/bin_steam.sh	2020-11-03 19:42:49.317085976 +0100
@@ -15,6 +15,10 @@
 bootstrapdir="$(dirname "$bootstrapscript")"
 
 export STEAMSCRIPT_VERSION=1.0.0.68
+# Make various games running in wine/proton happy
+if [ -z "$STEAM_DONT_MESS_WITH_PULSE" ]; then
+       export PULSE_LATENCY_MSEC=60
+fi
 
 # Set up domain for script localization
 export TEXTDOMAIN=steam
