diff -up steam-launcher/bin_steam.sh.1~ steam-launcher/bin_steam.sh
--- steam-launcher/bin_steam.sh.1~	2020-07-10 02:00:39.964414986 +0200
+++ steam-launcher/bin_steam.sh	2020-07-10 02:02:16.401246105 +0200
@@ -11,6 +11,10 @@ set -e
 # Get the full name of this script
 STEAMSCRIPT="$(cd "${0%/*}" && echo "$PWD")/${0##*/}"
 export STEAMSCRIPT
+export PATH="$HOME/.steam/bin32:$PATH"
+find "$HOME/.steam/root" "$HOME/.local/share/Steam" \( -name "libgcc_s.so*" -o -name "libstdc++.so*" -o -name "libxcb.so*" -o -name "libgpg-error.so*" \) -print -delete 2>/dev/null || :
+[ ! -e "$HOME/.steam/bin32" ] && mkdir -p "$HOME/.steam/bin32"
+[ ! -e "$HOME/.steam/bin32/tar" ] && ln -sf /usr/bin/gtar "$HOME/.steam/bin32/tar"
 bootstrapscript="$(readlink -f "$STEAMSCRIPT")"
 bootstrapdir="$(dirname "$bootstrapscript")"
 
@@ -209,4 +213,7 @@ fi
 cp "$LAUNCHSTEAMBOOTSTRAPFILE" "$LAUNCHSTEAMDIR/bootstrap.tar.xz"
 cd "$LAUNCHSTEAMDIR"
 
+# (tpg) use our own libraries
+find "$HOME/.steam/root" "$HOME/.local/share/Steam" \( -name "libgcc_s.so*" -o -name "libstdc++.so*" -o -name "libxcb.so*" -o -name "libgpg-error.so*" \) -print -delete 2>/dev/null || :
+
 exec "$LAUNCHSTEAMDIR/$STEAMBOOTSTRAP" "$@"
